package.json

```diff
@@ -237,6 +244,24 @@
                     "command": "extension.helmDepUp",
                     "group": "2_helm@99"
                 },
+                {
+                    "title": "Create Kubernetes resource",
+                    "when": "resourceLangId == yaml && config.vs-kubernetes.resource-context-menu",
+                    "command": "extension.vsKubernetesCreateFile",
+                    "group": "2_k8s"
```

Comment:
You're not specifying any ordering within this group.  Should we do that, e.g. Create, Apply, Delete?

Reply:
done.